enatega-multivendor-app/src/components/Main/TopBrands/TopBrands.js

```diff
@@ -30,6 +31,25 @@ function TopBrands(props) {
     }
   })
 
+  function loadingScreen() {
```

Comment:
This loading component is being used in multiple places so it's better to make a separate component for loading and pass children as in each screen children may differ.

Reply:
Its structure is different on multiple screens, so that's why I am keeping it as it was before being used in other components.